src/util-cpu.c

```diff
@@ -75,9 +76,15 @@ uint16_t UtilCpuGetNumProcessorsConfigured(void)
 
     return (uint16_t)nprocs;
 #elif OS_WIN32
-	long nprocs = -1;
-	const char* envvar = getenv("NUMBER_OF_PROCESSORS");
-	nprocs = (NULL != envvar) ? atoi(envvar) : 0;
+    int64_t nprocs = 0;
```

Comment:
63 bits -- `int64_t` seems large for the cpu count

Reply:
I think some of the OS specific APIs return `long`, so thats why that is used here. Converting it w/o down casting makes sense to me.